src/utils/src/Tags.c

```diff
@@ -75,9 +75,13 @@ STATUS packageTags(UINT32 tagCount, PTag pSrcTags, UINT32 tagsSize, PTag pDstTag
         curSize += nameSize + valueSize;
     }
 
+    // Round-up the allocation size to the 64 bit
+    curSize = ROUND_UP(curSize, 8);
```

Comment:
Should be HEAP_PACKED_SIZE()?

Reply:
No - these are always aligned unlike the heap